pkg/kubelet/kubelet.go

```diff
@@ -1833,6 +1833,12 @@ func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHand
 		factor = 2
 	)
 	duration := base
+	// Responsible for checking limits in resolv.conf
+	// The limits do not have anything to do with individual pods
+	if kl.dnsConfigurer != nil && kl.dnsConfigurer.ResolverConfig != "" {
```

Comment:
Nit: We could extract this into a function with a descriptive name (i.e. `performOnlyNecessaryResolvConfLimitsCheck`). That might be overengineering though... up to you :)

Reply:
Since there are only a few lines and it is called once, probably we don't need to extract it.